docs/keosd/introduction.md

```diff
@@ -1,7 +1,5 @@
-## Introduction
+`keosd` is a key manager for store private keys and sign transactions created by `cleos`. It providdes a secure key storage medium so that when the private keys is stored in a wallet file , it will be encrypted at rest. Only when a wallet is unlocked with the wallet password, `cleos` can request `keosd` to sign a transaction with these private keys. `keosd` provides support for hardware-based wallets as well such as Secure Encalve and YubiHSM
 
-What is keosd. What is its goal and scope
+`keosd` is intended to be used by EOSIO developers and power users
```

Comment:
I don't think we want to frame `keosd` as a tool for power users, only developers really.  Basically, it has not had the scrutiny applied to it would need to hold keys for large amounts of real value.   It may be up to that task but we certainly do not want to position it as such.